Source/Scene/ParticleSystem.js

```diff
@@ -40,36 +44,35 @@ define([
      *
      * @param {Object} [options] Object with the following properties:
      * @param {Boolean} [options.show=true] Whether to display the particle system.
-     * @param {ParticleSystem~applyForce[]} [options.forces] An array of force callbacks.
+     * @param {ParticleSystem~updateCallback} [options.updateCallback] The callback to an update function used for every particle in this system.
      * @param {ParticleEmitter} [options.emitter=new CircleEmitter(0.5)] The particle emitter for this system.
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix that transforms the particle system from model to world coordinates.
      * @param {Matrix4} [options.emitterModelMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix that transforms the particle system emitter within the particle systems local coordinate system.
-     * @param {Color} [options.startColor=Color.WHITE] The color of a particle when it is born.
-     * @param {Color} [options.endColor=Color.WHITE] The color of a particle when it dies.
-     * @param {Number} [options.startScale=1.0] The scale of the particle when it is born.
-     * @param {Number} [options.endScale=1.0] The scale of the particle when it dies.
-     * @param {Number} [options.rate=5] The number of particles to emit per second.
+     * @param {Number} [options.emissionRate=5] The number of particles to emit per second.
      * @param {ParticleBurst[]} [options.bursts] An array of {@link ParticleBurst}, emitting bursts of particles at periodic times.
      * @param {Boolean} [options.loop=true] Whether the particle system should loop it's bursts when it is complete.
-     * @param {Number} [options.speed] Sets the minimum and maximum speed in meters per second
+     * @param {Number} [options.scale] Sets the start and end scales.
```

Comment:
I'm not sure I understand why we are making scale the main variable here and letting start and endScale be the secondary variables when scale is not actually used by the class itself anywhere else but in the constructor.

I'm currently describing what the constructor is actually doing with the variable.

Is this change just for readability?

Reply:
I mean just in terms of making the documentation clear.

Reply:
would a line like this work for options.scale? [and the same formatting of this line for other start/end and min/max overriding variable documentation]

If set, overrides the startScale and endScale inputs with this value.

Reply:
I think it may be best the other way around - When you can set the `scale` property that's what `start` and `end` default to. Then if those are set, they will override that scale value. `start` and `end` should not specify default values in the docs. `scale` _should_ specify the default. 

Does that make sense?

Reply:
ohhhh gotcha makes sense now - i'll do that